po/pt-br.po

```diff
@@ -0,0 +1,381 @@
+msgid ""
+"added more translations (tamil, hungary, catalan) and update russian "
+"translations"
+msgstr ""
+
+#: data/com.vixalien.sticky.appdata.xml.in.in:101
+msgid ""
+"you can now choose to use dark or light mode manually instead of always "
+"using the system theme in the all notes window"
+msgstr ""
+
+#: data/com.vixalien.sticky.appdata.xml.in.in:108
+msgid "this is unexpectedly a large release"
+msgstr ""
+
+#: data/com.vixalien.sticky.appdata.xml.in.in:110
+msgid "new icon"
+msgstr ""
+
+#: data/com.vixalien.sticky.appdata.xml.in.in:111
+msgid "added translations (french, russian, italian, dutch, ocitan)"
+msgstr ""
+
+#: data/com.vixalien.sticky.appdata.xml.in.in:112
+msgid "bug fixes"
+msgstr ""
+
+#: data/com.vixalien.sticky.appdata.xml.in.in:118
+msgid "this is mostly a cleanup release"
+msgstr ""
+
+#: data/com.vixalien.sticky.appdata.xml.in.in:120
+msgid "added translation support (starting with french)"
+msgstr ""
+
+#: data/com.vixalien.sticky.appdata.xml.in.in:121
+msgid ""
+"apply tag to location under the cursor when nothing is selected. this means "
+"that for example if you click Ctrl+B while there's nothing selected, the new "
+"content you'll write will come out as bold"
+msgstr ""
+
+#: data/com.vixalien.sticky.appdata.xml.in.in:124
+msgid "fixed builds"
+msgstr ""
+
+#: data/com.vixalien.sticky.appdata.xml.in.in:125
+msgid "restructured/cleaned source"
+msgstr ""
+
+#: data/com.vixalien.sticky.appdata.xml.in.in:131
+msgid "First beta release!"
+msgstr ""
+
+#: data/com.vixalien.sticky.gschema.xml.in:16
+msgid "Note window default height"
+msgstr ""
+
+#: data/com.vixalien.sticky.gschema.xml.in:17
+msgid "The default height of a new sticky note"
+msgstr ""
+
+#: data/com.vixalien.sticky.gschema.xml.in:21
+msgid "Note window default width"
+msgstr ""
+
+#: data/com.vixalien.sticky.gschema.xml.in:22
+msgid "The default width of a new sticky note"
+msgstr ""
+
+#: data/com.vixalien.sticky.gschema.xml.in:26
+msgid "Default note style"
+msgstr ""
+
+#: data/com.vixalien.sticky.gschema.xml.in:27
+msgid "The default style assigned to a new note"
+msgstr ""
+
+#: data/com.vixalien.sticky.gschema.xml.in:31
+msgid "Confirm note deletion"
+msgstr ""
+
+#: data/com.vixalien.sticky.gschema.xml.in:32
+msgid "Whether to confirm note deletion"
+msgstr ""
+
+#: data/com.vixalien.sticky.gschema.xml.in:36
+msgid "The selected color scheme"
+msgstr ""
+
+#: data/com.vixalien.sticky.gschema.xml.in:37
+msgid "The selected color scheme. 0 = follow system style, 1 = light, 2 = dark"
+msgstr ""
+
+#: data/com.vixalien.sticky.gschema.xml.in:43
+msgid "Show all notes"
+msgstr ""
+
+#: data/com.vixalien.sticky.gschema.xml.in:44
+msgid "Whether to show the all notes window or note"
+msgstr ""
+
+#: data/ui/help-overlay.ui:11
+msgctxt "shortcut window"
+msgid "General"
+msgstr ""
+
+#: data/ui/help-overlay.ui:14
+msgctxt "shortcut window"
+msgid "Show Shortcuts"
+msgstr "Atalhos de teclado"
+
+#: data/ui/help-overlay.ui:20
```

Comment:
Would you mind translating verbs from help-overlay.ui using conjugation in third person present tense? For instance, Quit = Sai, instead of Sair. It is not wrong as is now, just that I try to keep all GTK apps (mostly GNOME projects) translated that way.